### DISCUSSION TITLE
Suggestion - Media filter for Home feed

### DISCUSSION THREAD

**Discussion by sygninx** [mastodon/mastodon#20613] [General]
On my desktop (running the advanced web interface), my Local timeline has an option to show/hide only posts with Media.

My Home timeline has the option to show/hide boosts & replies, which is handy. If the show/hide option for Media can be added, it would be great as an additional filter. 

(I'm not a dev; I was asking about this on my account and someone suggested I request this as a feature here)

**Comment by ineffyble** [comment]
I've created an issue/suggestion here: https://github.com/mastodon/mastodon/issues/20619
